src/include/btree_inline.h

```diff
@@ -1646,17 +1663,35 @@ __wt_page_del_visible(WT_SESSION_IMPL *session, WT_PAGE_DELETED *page_del, bool
     /* We discard page_del on transaction abort, so should never see an aborted one. */
     WT_ASSERT(session, page_del->txnid != WT_TXN_ABORTED);
 
+    if (!__wt_txn_visible(session, page_del->txnid, page_del->timestamp))
+        return (false);
+
+    /*
+     * Check whether truncate transaction is committed or not. In case of prepared transactions,
+     * truncate operation is committed in two phases. In the first phase prepared state is set to
```

Comment:
I know you got this from Vamsi's change but can we update this comment to be "the truncate operation is committed in two phases." Same with the other one.